### PR TITLE
Kaitie - Amplitude Event for changing rubric evaluation

### PR DESCRIPTION
**Pull request by kobryan0619**
This event tracks every time a teach changes the evaluation for a Evidence Level Selected by a teacher evaluating student work.

This is my first time setting up an amplitude event which is why I am seeking out feedback for this TINY PR - looking to make sure it all looks good before doing the rest of the events. 

Here's what it looks like in the console:

image

## Links

[Jira Ticket](https://codedotorg.atlassian.net/jira/software/projects/AITT/boards/70/backlog?selectedIssue=AITT-195)

## Testing story

I modified the test for EvidenceLevels as this is where the call to the function that triggers the event takes place. 

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/rubrics/LearningGoal.jsx
Comment (kobryan0619) [inline apps/src/templates/rubrics/LearningGoal.jsx:149]: This allows us to pass through the other reporting data from this learning goal such as the unit, course, and level

File: apps/src/templates/rubrics/LearningGoal.jsx
Comment (kobryan0619) [inline apps/src/templates/rubrics/LearningGoal.jsx:153]: If this is the first time evaluating the student work, the `understandingLevel.current` will be -1.

**Review by bethanyaconnor — APPROVED** [review]
LGTM!

I normally tag @tess323 on Amplitude PRs but don't wait for Tess' approval to merge (Tess -- let us know if you want a different process!)

**Pull request merged**
